contributors/guide/contributor-cheatsheet/README-es.md

```diff
@@ -0,0 +1,396 @@
+    - [Agregar Upstream](#agregar-upstream)
+    - [Mantener tu Fork Sincronizado](#mantener-tu-fork-sincronizado)
+    - [Combinar Commits (Squashing Commits)](#squashing-commits)
+   
+---
+## Recursos Útiles
+
+### Empezando
+
+- [Curso para Colaboradores] - **NUEVO** - El curso de E-Learning para colaboradores de Kubernetes!
+- [Guía para Colaboradores] - Guía sobre cómo comenzar a contribuir al Proyecto
+  Kubernetes.
+- [Guía del Desarrollador] - Guía para contribuir código directamente al Proyecto
+  Kubernetes.
+- [Guía de Seguridad y Divulgación] - Guía para informar vulnerabilidades
+  y el proceso de liberación de seguridad.
+
+### SIGs y Otros Grupos
+
+- [Lista Maestra de Grupos][sigs]
+
+### Comunidad
+
+- [Calendario] - Visualiza todos los eventos de la Comunidad de Kubernetes (reuniones de SIG/WG,
+  eventos, etc.).
+- [kubernetes-dev] - La lista de correo de desarrollo de Kubernetes.
+- [Foro de Kubernetes] - Foro oficial de Kubernetes.
+- [Canales de Slack] - Slack oficial de Kubernetes.
+- [Stack Overflow] - Un lugar para hacer preguntas como usuario final de Kubernetes.
+- [Canal de YouTube] - Canal oficial de la comunidad de Kubernetes.
+
+### Flujo de Trabajo
+
+- [Panel de Gubernator] - Visualiza las Solicitudes de Extracción entrantes y salientes que requieren
+  tu atención.
+- [Prow] - Sistema de CI/CD de Kubernetes.
+- [Tide] - Plugin de Prow que administra fusiones y pruebas. [Panel de Tide]
+- [Comandos de Bots] - Comandos utilizados para interactuar con los Bots de Kubernetes (ejemplos:
+- `/cc`, `/lgtm` y `/retest`).
+- [GitHub Labels] - Lista de labels utilizados en todo el Proyecto Kubernetes.
+- [Búsqueda de Código de Kubernetes], mantenida por [@dims].
+
+### Pruebas
+
+- [Prow] - Sistema de CI/CD de Kubernetes.
+- [Test Grid] - Visualiza pruebas históricas y su información asociada.
+- [Panel de Triage] - Agrega problemas similares para una mejor
+  resolución de problemas.
+
+### Alias de Correo Importantes
+
+- community@kubernetes.io - Envía un correo a alguien del equipo de la comunidad
+  (SIG Contributor Experience) acerca de un problema de comunidad.
+- conduct@kubernetes.io - Contacta al comité de Código de Conducta,
+  lista de correo privada.
+- github@kubernetes.io - Envía un correo al [Equipo de Administración de GitHub] de forma privada,
+  para asuntos sensibles.
+- steering@kubernetes.io - Envía un correo al comité directivo. Dirección pública
+  con archivo público.
+- steering-private@kubernetes.io - Envía un correo al comité directivo de forma privada,
+  para asuntos sensibles.
+- social@cncf.io - Contacta al equipo social de la CNCF; blog, cuenta de Twitter
+  y otras propiedades sociales.
+
+### Otros Enlaces Útiles
+
+- [Estadísticas de Desarrolladores] - Visualiza estadísticas de desarrolladores para todos los
+  proyectos gestionados por la CNCF.
+- [Lanzamiento de Parches de Kubernetes] - Programación e información de contacto del equipo
+  para lanzamientos de parches de Kubernetes.
+
```

Comment:
https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md

Reply:
Yes that would be great